### PR TITLE
CVE-2013-0891 and CVE-2014-1732

### PR DESCRIPTION
**Pull request by AndrewDelposen**
Initial pull request for setting up for future commits

### CODE REVIEW

File: cves/CVE-2013-0891.yml [hunk 1]
```diff
@@ -52,9 +55,11 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-- :commit: 0ed41ef78a807f5d195b070bce7c433fcf6f5cb3
+- :commit: 811f3cdb481f280a40ca6c3a4df38c78007fc654
```
Comment (mesternefeld) [inline cves/CVE-2013-0891.yml]: Definitely would keep the other commit as well as it is part of getting to the fix

File: cves/CVE-2013-0891.yml [hunk 2]
```diff
@@ -75,9 +80,14 @@ unit_tested:
...
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
-  fix: 
+  answer: |
+    From looking at the files affected, and the code specifically, there seems to be no
+    automated tests, nor any tests related the files as a whole.
+    
+    The code review and bug tracker did show that the code was being run against the
+    server as continuous test integration, which is why I'm saying "true" for code
```
Comment (mesternefeld) [inline cves/CVE-2013-0891.yml]: Love the explanation here!

File: cves/CVE-2013-0891.yml [hunk 3]
```diff
@@ -52,9 +55,11 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-- :commit: 0ed41ef78a807f5d195b070bce7c433fcf6f5cb3
+- :commit: 811f3cdb481f280a40ca6c3a4df38c78007fc654
   :note: ''
-vccs: []
+vccs: 
+- :commit: 0ed41ef78a807f5d195b070bce7c433fcf6f5cb3
```
Comment (mesternefeld) [inline cves/CVE-2013-0891.yml]: This is not the vcc but rather part of the fix. I'd keep digging back a few commits

File: cves/CVE-2014-1732.yml [hunk 4]
```diff
@@ -4,14 +4,14 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
```
Comment (mesternefeld) [inline cves/CVE-2014-1732.yml]: I found this vulnerability interesting with how it affects older machines and was fixed by a simple observer pattern. 5 upvotes from me! 🎉

File: cves/CVE-2013-0891.yml (hunk 3 above)
Comment (blackpan2) [reply cves/CVE-2013-0891.yml]: Looking at this [diff](https://src.chromium.org/viewvc/chrome/trunk/src/webkit/blob/blob_url_request_job.cc?r1=163245&r2=179901&pathrev=179901), this seems to be the VCC.

SVN commit 163245

File: cves/CVE-2013-0891.yml [hunk 5]
```diff
@@ -149,8 +162,10 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies: 
-    note: 
+    applies: true
+    note: |
+      The solution to this vulnerability including checking the cast values and limits
```
Comment (blackpan2) [inline cves/CVE-2013-0891.yml]: 👍

File: cves/CVE-2014-1732.yml (hunk 4 above)
Comment (blackpan2) [reply cves/CVE-2014-1732.yml]: Observer is the best +2

**Pull request merged**